core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1416,6 +1416,17 @@ package object config {
       .stringConf
       .createWithDefault("zstd")
 
+  private[spark] val MAP_STATUS_OUTPUT_CHUNK_SIZE =
```

Comment:
Do we really need to make this a config option?

Reply:
According to @srowen and @mridulm 's advice, shall we remove this, @kazuyukitanimura ?

Reply:
Thanks, removed.